styles/Vocab/Plone/accept.txt

```diff
@@ -1,10 +1,22 @@
+async
+api 
+backend
+css
+http 
+https 
+JavaScript
+LTS 
+npm
+nvm 
```

Comment:
"npm" and "nvm" are good here. They are are the actual and proper names of software. "NPM" and "NVM" would be wrong. These are exceptions to the rule for acronyms.